botfront/imports/api/graphql/config/configResolver.js

```diff
@@ -0,0 +1,31 @@
+import yaml from 'js-yaml';
+import { Credentials, Endpoints } from './config.models.js';
+
+export default {
```

Comment:
It seems to me that it should be a more generic `getProject(s)` query where credentials/endpoints are fields

```
getProject(projectId: "bf') {
   credentials
   endpoints
}
```

Reply:
can this be in a further iteration?